### ISSUE TITLE
LazyVim 14.X changed scope lines to white

### ISSUE THREAD

**Issue by pfassina** [aktersnurra/no-clown-fiesta.nvim#49]
The newest LazyVim version changed scope lines from light grey to full bright white, which is distracting and potentially against the theme intent.

I'm trying to pinpoint the actual change, but I think it might be related to the new snacks plugins used by the newest lazyvim version. Maybe snacks.

![image](https://github.com/user-attachments/assets/e7ea7703-719c-482b-acdf-7e4c7d08a16b)

**Comment by pfassina** [comment]
I've found a `snacks.lua` config on tokyonight, which might be a path to solve the problem:
https://github.com/folke/tokyonight.nvim/blob/main/lua/tokyonight/groups/snacks.lua

**Comment by aktersnurra** [comment]
Yeah, not visually pleasing. Most likely he change the default highlight group name.

Apparently, you can set them yourself via the config here:

```lua
    hl = "SnacksIndent", ---@type string|string[] hl groups for indent guides
    -- can be a list of hl groups to cycle through
    -- hl = {
    --     "SnacksIndent1",
    --     "SnacksIndent2",
    --     "SnacksIndent3",
    --     "SnacksIndent4",
    --     "SnacksIndent5",
    --     "SnacksIndent6",
    --     "SnacksIndent7",
    --     "SnacksIndent8",
    -- },
  },
  --
```

So, either configure your own highlight groups and use them in the config, or choose something from this color scheme, e.g. `Comment`.
